src/System.Globalization.Calendars/tests/CalendarHelpers.cs

```diff
@@ -377,8 +378,15 @@ public static void TwoDigitYearMax_Invalid(Calendar calendar)
         [MemberData(nameof(DateTime_TestData))]
         public static void GetEra_Invalid(Calendar calendar, DateTime dt)
         {
-            // TODO: This fails for HebrewCalendar, TaiwanLunisolarCalendar and JapaneseLunisolarCalendar
-            // Assert.Throws<ArgumentOutOfRangeException>(() => calendar.GetEra(dt));
+        	// JapaneseCalendar throws on Unix (ICU), but not on Windows
+            if ((calendar is JapaneseCalendar && PlatformDetection.IsWindows) || calendar is HebrewCalendar || calendar is TaiwanLunisolarCalendar || calendar is JapaneseLunisolarCalendar)
```

Comment:
do we understand why Linux is throwing? maybe we have a bug? also I don't like the to special case some calendars like this. do we understand why we are special case these calendars here?